peps/pep-0667.rst

```diff
@@ -371,6 +371,70 @@ C API
 As with all functions that return a borrowed reference, care must be taken to
 ensure that the reference is not used beyond the lifetime of the object.
 
+Impact on ``exec()``
+====================
+
+Even though this PEP does not modify ``exec()`` directly, the semantic change
+to ``locals()`` impacts the behavior of ``exec()``.
+
+``exec(object)`` has been equivalent to ``exec(object, globals(), locals())``
+for a long time, but it's not officially documented. We plan to keep this
```

Comment:
```suggestion
for a long time, but it was not officially documented. We plan to keep this
```

Reply:
I reworded this paragraph (the old behaviour was documented, but the docs aren't especially clear about it)